packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -262,7 +262,7 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
     }
 
     protected renderReplaceFieldToggle(): React.ReactNode {
-        const toggle = <span className={`fa fa-caret-${this.showReplaceField ? 'down' : 'right'}`}></span>;
+        const toggle = <span title='Toggle Replace' className={`fa fa-caret-${this.showReplaceField ? 'down' : 'right'}`}></span>;
         return <div
```

Comment:
should we add `title` to this div instead ? when testing i noticed that i had to mouse-over that caret to get tooltip, while in vsCode mouse-over-ing the button makes the tooltip visible.

Reply:
thanks @elaihau, updated! :)

Reply:
What about the tooltip over the "Search" and "Replace" text box , should we implement a tooltip there as well like vscode?

Reply:
sure, updated